conf/messages.en

```diff
@@ -581,6 +581,14 @@ wasSettlorsAddressUKYesNo.error.required = Select yes if the settlor’s last kn
 deceasedSettlorAnswer.title = Check settlor’s details
 deceasedSettlorAnswer.heading = Check settlor’s details
 
+agentName.title = What is the name of the agency registering this trust?
+agentName.heading = What is the name of the agency registering this trust?
+agentName.checkYourAnswersLabel = What is the name of the agency registering this trust?
+agentName.fullname = Your full name
+agentName.hint = What is the name of the agency registering this trust?
+agentName.error.required = Enter the agent’s full name
+agentName.error.length = Agent’s full name must be 56 characters or less
```

Comment:
@codebysachin I think it should be agency' name must be instead of agent full name

Reply:
done with the changes